fem/bilinearform.hpp

```diff
@@ -95,6 +95,13 @@ class BilinearForm : public Matrix
    Array<BilinearFormIntegrator*> bfbfi;
    Array<Array<int>*>             bfbfi_marker; ///< Entries are not owned.
 
+   /// Set of normal derivative interior face Integrators to be applied.
+   Array<BilinearFormIntegrator*> ndfbfi;
```

Comment:
I'm not sure we want to distinguish between "interior face Integrators" and "normal derivative interior face Integrators" here.

Reply:
That makes sense. I think now I have a better sense of what the interface needs to enable merging these into one integrator.